.github/workflows/callgrind_benchmarks.yml

```diff
@@ -0,0 +1,179 @@
+on:
```

Comment:
I didn't look closely at this. Probably someone more familiar with our CI should peek at this.

You could help by saying what has to be different between this and the existing `benchmarks please` flow, perhaps.

Reply:
Specifically, you have to comment `callgrind please`.